package.json

```diff
@@ -28,7 +28,7 @@
     "lint": "eslint lib test",
     "test": "npm run lint && karma start config/karma.js --single-run",
     "test:watch": "karma start config/karma.js",
-    "test:integration": ". ./.env && bundle exec rake sauce:spec test_files=spec",
```

Comment:
We could leave this in, and just have travis `touch .env` so it doesn't error when it doesn't exist.